internal/testing/setup/setup.go

```diff
@@ -127,7 +128,9 @@ func NewAWSSession2(ctx context.Context, t *testing.T, region string) (sess *ses
 			}
 		}
 	} else { // replay
+		start := time.Now()
 		rep, err := httpreplay.NewReplayer(path)
+		t.Logf("starting http replayer took %s", time.Since(start))
```

Comment:
I don't think this will show up in Travis output ordinarily, since we don't run with `-v`. Perhaps use the standard logging package?